frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/create-local-volume-set/body.tsx

```diff
@@ -0,0 +1,321 @@
+            >
+              <Tooltip
+                content={
+                  !validMaxDiskSize
+                    ? 'Please enter a positive Integer'
+                    : 'Please enter a value greater than or equal to min disk size'
+                }
+                isVisible={!validMaxDiskSize || (invalidMinGreaterThanMax && activeMaxDiskSize)}
+                trigger="manual"
+              >
+                <TextInput
+                  type={TextInputTypes.text}
+                  id="create-lvs-max-disk-size"
+                  value={state.maxDiskSize}
+                  validated={
+                    !validMaxDiskSize || (invalidMinGreaterThanMax && activeMaxDiskSize)
+                      ? 'error'
+                      : 'default'
+                  }
+                  className="odf-create-lvs__disk-input"
+                  onFocus={() => setMaxActiveState(true)}
+                  onBlur={() => setMaxActiveState(false)}
+                  onChange={(value) => formHandler('maxDiskSize', value)}
+                />
+              </Tooltip>
+            </FormGroup>
+            <Dropdown
+              id="create-lvs-disk-size-unit-dropdown"
+              items={diskSizeUnitOptions}
+              title={diskSizeUnitOptions[state.diskSizeUnit]}
+              selectedKey={state.diskSizeUnit}
+              onChange={(unit: string) => formHandler('diskSizeUnit', unit)}
+            />
+          </div>
+        </FormGroup>
+        <FormGroup
+          label={t('ceph-storage-plugin~Maximum Disks Limit')}
+          fieldId="create-lvs-max-disk-limit"
+        >
+          <p className="help-block odf-create-lvs__max-disk-limit-help-text--margin">
+            {t(
+              'ceph-storage-plugin~Disks limit will set the maximum number of PVs to create on a node. If the field is empty we will create PVs for all available disks on the matching nodes.',
+            )}
+          </p>
+          <Tooltip
+            content="Please enter a positive Integer"
+            isVisible={!validMaxDiskLimit}
+            trigger="manual"
+          >
+            <TextInput
+              type={TextInputTypes.text}
+              id="create-lvs-max-disk-limit"
+              value={state.maxDiskLimit}
+              validated={validMaxDiskLimit ? 'default' : 'error'}
+              className="odf-create-lvs__disk-input"
+              onChange={(maxLimit) => formHandler('maxDiskLimit', maxLimit)}
+              placeholder={t('ceph-storage-plugin~All')}
+            />
+          </Tooltip>
+        </FormGroup>
+      </ExpandableSection>
+    </>
+  );
+};
+
+type LocalVolumeSetBodyProps = {
+  state: WizardState['createLocalVolumeSet'];
+  dispatch: WizardDispatch;
+  storageClassName: string;
+  diskModeOptions?: { [key: string]: string };
+  deviceTypeOptions?: { [key: string]: string };
+  allNodesHelpTxt?: string;
+  lvsNameHelpTxt?: string;
+  taintsFilter?: (node: NodeKind) => boolean;
```

Comment:
```suggestion
  taintsFilter?: (node: NodeKind, Taint t) => boolean;
```
Could be changed to a pure function for better usabilty

Reply:
Implementation details involved. A lot of code will be affected.
I just copied it as it is now.